irr/src/OpenGLES2/Driver.cpp

```diff
@@ -86,10 +86,8 @@ void COpenGLES2Driver::initFeatures()
 		TextureFormats[ECF_R8G8B8] = {GL_RGB, GL_RGB, GL_UNSIGNED_BYTE};
 		TextureFormats[ECF_A8R8G8B8] = {GL_RGBA, GL_RGBA, GL_UNSIGNED_BYTE, CColorConverter::convert_A8R8G8B8toA8B8G8R8};
 
-		if (FeatureAvailable[IRR_GL_EXT_texture_format_BGRA8888])
+		if (FeatureAvailable[IRR_GL_EXT_texture_format_BGRA8888] || FeatureAvailable[IRR_GL_APPLE_texture_format_BGRA8888])
 			TextureFormats[ECF_A8R8G8B8] = {GL_BGRA, GL_BGRA, GL_UNSIGNED_BYTE};
-		else if (FeatureAvailable[IRR_GL_APPLE_texture_format_BGRA8888])
-			TextureFormats[ECF_A8R8G8B8] = {BGRA8_EXT, GL_BGRA, GL_UNSIGNED_BYTE};
```

Comment:
comparing https://registry.khronos.org/OpenGL/extensions/EXT/EXT_texture_format_BGRA8888.txt and https://registry.khronos.org/OpenGL/extensions/APPLE/APPLE_texture_format_BGRA8888.txt, looks like the APPLE version (contrary to the EXT version) has always defined `GL_BGRA8_EXT`, so we can rely on it

why did you make this change, making this code path also rely on the workaround?